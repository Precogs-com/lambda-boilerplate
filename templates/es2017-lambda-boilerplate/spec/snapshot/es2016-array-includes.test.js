const { transform } = require('./utils');

it('array literal expression', () => {
    const code = transform('[1, 2, 3].includes(1)');
    expect(code).toMatchSnapshot();
});

it('Array() constructor literal expression', () => {
    const code = transform('Array(1, 2, 3).includes(1)');
    expect(code).toMatchSnapshot();
});
