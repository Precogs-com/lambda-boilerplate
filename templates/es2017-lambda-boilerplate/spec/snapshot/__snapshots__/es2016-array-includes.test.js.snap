// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Array() constructor literal expression 1`] = `
"\\"use strict\\";

Array(1, 2, 3).indexOf(1) !== -1;"
`;

exports[`array literal expression 1`] = `
"\\"use strict\\";

[1, 2, 3].indexOf(1) !== -1;"
`;
